src/components/ui/DataGrid/BulkActionDataGrid.svelte

```diff
@@ -214,6 +215,9 @@
           </ContextMenu.Item>
         </div>
       {/if}
+
+      <slot name="context-menu-alt" />
```

Comment:
I believe it was agreed upon to not add this extra slot, but to add the new content in the actual implementation of the slot.